monai/transforms/utils_pytorch_numpy_unification.py

```diff
@@ -104,8 +104,10 @@ def percentile(
     elif any(q < 0) or any(q > 100):
```

Comment:
Hi @wyli , thanks for the fix. It is not part of the fix but I think here q may also support list. 
For example, this works with numpy:
```python
np.percentile(np.random.rand(10,1), q=[0.5,99.5])
```
But fails with percentile:
```python
percentile(np.random.rand(10,1), q=[0.5,99.5])
```
```bash
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/workspace/monai/monai-auto3d-1.0/monai/transforms/utils_pytorch_numpy_unification.py", line 104, in percentile
    elif any(q < 0) or any(q > 100):
TypeError: '<' not supported between instances of 'list' and 'int'
```

Reply:
added the support, thanks!